amex-page.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>AMEX</title>
+        <style>
+            .container {
+                max-width: 1200px;
+            }
+            .row {
+                clear: float;
+            }
+            .column {
+                width: 25%;
+                float: left;
+            }
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            .col-2 {
+                width: 10%;
+                float: left;
+            }
+            .hero-image {
+                background-image: url(fistbump-over-desk.jpg);
```

Comment:
Since `fistbump-over-desk.jpg` is a string, it requires quotations around it.
Strings also require quotations around them.